src/stores/studyStore.ts

```diff
@@ -175,10 +178,6 @@ export const useStudyStore = defineStore('study', () => {
       });
   }
 
-  function exportStudyCalendar(studyId: number): void {
-    window.open(`api/v1/studies/${studyId}/calendar.ics`);
```

Comment:
This means that we also have a corresponding controller in the studymanager-backend. That one should be removed too!